addons/account/models/ir_actions_report.py

```diff
@@ -44,10 +44,13 @@ def _render_qweb_pdf_prepare_streams(self, report_ref, data, res_ids=None):
                 }
         return collected_streams
 
+    def _is_invoice_report(self, report_ref):
+        return True if self._get_report(report_ref).report_name in ('account.report_invoice_with_payments', 'account.report_invoice') else False
```

Comment:
It seems your rebase changes this part of code for the original code you made. Could you please simplify as requested before? Thank you :)